extensions/positron-python/python_files/positron/positron_ipykernel/tests/test_variables.py

```diff
@@ -226,6 +226,30 @@ def test_list_1000(shell: PositronShell, variables_comm: DummyComm) -> None:
     assert variables[999].get("display_name") == "var999"
 
 
+def test_list_falls_back_on_variable_error(
```

Comment:
This tests the fallback variable creation code path to ensure we don't break it again.